### ISSUE TITLE
Is this still maintained?

### ISSUE THREAD

**Issue by jcbhmr** [vincentdoerig/latex-css#54]
![image](https://user-images.githubusercontent.com/61068799/216159340-c03a2ab0-7e41-4d79-b92a-5b7e8dd7227a.png)

**Comment by vincentdoerig** [comment]
Yes, this project is still being maintained, but I do have a lot on my plate at the moment. I will do my best to review the open PRs by the end of this month.

Cheers!

**Comment by jcbhmr** [comment]
Awesome! 🥇 Thanks for the confirmation! ❤️
